packages/library-base/src/commonMain/kotlin/io/realm/MutableRealm.kt

```diff
@@ -79,9 +79,10 @@ public interface MutableRealm : TypedRealm {
     ): RealmQuery<T>
 
     /**
-     * Deletes the object from the underlying Realm. Only live objects can be deleted.
+     * Deletes object or objects of a query or result.
```

Comment:
```suggestion
     * Delete objects from the underlying Realm. [RealmObject], [RealmList], [RealmQuery] and [RealmResults] can be deleted this way.
```

The old description was a bit hard to parse for me...especially `objects of a query or result.`. Tried to reformulate.

Reply:
In Realm Java we didn't have a `deleteAllFromRealm` method on `RealmQuery`, but I don't think it is harmful and given how our query API is structured it probably makes sense 👍